source/index.ts

```diff
@@ -285,6 +287,12 @@ function createMainWindow(): BrowserWindow {
 	mainWindow = createMainWindow();
 	tray.create(mainWindow);
 
+	// TODO: Reload mainWindow instead as soon as #712 is fixed
+	if (!(await isOnline()) && (await pWaitFor(isOnline), {interval: 1000})) {
```

Comment:
1. There's a bug – the options object with `interval` isn't actually passed into the `pWaitFor` 😉
2. It might be more readable to move the second await inside the block:

```suggestion
	if (!await isOnline()) {
		await pWaitFor(isOnline, {interval: 1000});
```

Reply:
1. I somehow missed that from too many parenthesis. 2. That's the reason i missed nr. 1

Reply:
You can also get rid of the extra parens around `await isOnline()` (see the previous suggestion). 🙂

Reply:
@CvX The linter won't really let me do that inside an if condition

Reply:
Ah, my bad, some linter errors don't show up in my text editor due to a bug… 😓